modules/core/src/main/java/org/apache/synapse/core/axis2/Axis2SynapseEnvironment.java

```diff
@@ -217,6 +222,37 @@ public void setSeverDebugMode(ServerContextInformation contextInformation) {
         }
     }
 
+    /**
+     * This method is to set the unit test mode is enabled.
+     * unit test message context and environment initializes
```

Comment:
Shall we correct the javadoc?

Reply:
Fixed in https://github.com/wso2/wso2-synapse/pull/1452